pyproject.toml

```diff
@@ -1,11 +1,11 @@
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "viz-python-lib"
-version = "0.1.0"
-description = "python library for VIZ blockchain"
+version = "0.2.0"
```

Comment:
I suggest to bump to `1.0.0` as python version dep is increased, and maintain semantic versioning further.